engine/src/rgbmatrix.cpp

```diff
@@ -242,6 +242,10 @@ void RGBMatrix::setAlgorithm(RGBAlgorithm *algo)
                     m_properties.take(it.key());
                 }
             }
+
+            QVector<uint> colors = script->rgbMapGetColors();
```

Comment:
After setting all the properties, the default colors should be loaded and displayed in the UI. If there are no default colors (and `rgbMapGetColors` returns an empty array), we default to what `RGBMatrix` defines as default (1 color: red).